IPs/RSKIP176.md

```diff
@@ -0,0 +1,115 @@
+# Fast Bridge
+|RSKIP          |           |
+| :------------ |:-------------|
+|**Title**      |Fast Bridge |
```

Comment:
Maybe it would be better to specify which bridge "BTC/RSK" ? Now that we also have the Eth->Rsk bridge.